user-guide/Standard_Apps/EPM/EPM_D-DOCSIS/D-DOCSIS_deployment.md

```diff
@@ -2,7 +2,22 @@
 uid: D-DOCSIS_deployment
 ---
 
-# EPM D-DOCSIS deployment
+# EPM D-DOCSIS Deployment
+
+### Upgrade
+
+To deploy and upgrade the D-DOCSIS branch of the EPM Solution in an automated fashion:
+
+1. Make sure the **latest DataMiner feature release** version is installed.
+
+1. Download the latest EPM D-DOCSIS upgrade installation package and place it on the server.
```

Comment:
Where can the user download the D-DOCSIS package? We should add a link or a reference to make this clear.

Reply:
Hi Marieke,
Currently the only place where the package can be downloaded is from [SVN](https://svn.skyline.be/!/#SystemEngineering/view/head/InstallPackages/Generic/DataMiner%20Solutions/EPM-D-DOCSIS). From speaking to a colleague, the plan is to make the package accessible to the intended customer through Sharepoint, and that ideally they would be downloadable through GitHub.

Reply:
That link is only available internally at Skyline, right? In that case, I'd propose to keep this step a little more vague for now:

```suggestion
1. Place the latest EPM D-DOCSIS upgrade installation package on the server.
```